From bd208aa992eee2f38706d3b72b1c743738fa4312 Mon Sep 17 00:00:00 2001
From: "h.udo" <hudokkow@gmail.com>
Date: Wed, 6 Jan 2016 15:21:14 +0000
Subject: [PATCH] [cmake] Update to p8-platform

---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cdceaeb..b4e0c5d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,10 +8,10 @@ enable_language(CXX)
 
 find_package(kodi REQUIRED)
 find_package(kodiplatform REQUIRED)
-find_package(platform REQUIRED)
+find_package(p8-platform REQUIRED)
 
 include_directories(${kodiplatform_INCLUDE_DIRS}
-                    ${platform_INCLUDE_DIRS}
+                    ${p8-platform_INCLUDE_DIRS}
                     ${KODI_INCLUDE_DIR}
                     ${PROJECT_SOURCE_DIR}/src)
 
@@ -66,7 +66,7 @@ endif(WIN32)
 # Make sure that CMake adds the headers to the MSVC project
 list(APPEND ARGUSTV_SOURCES ${ARGUSTV_HEADERS} ${RESOURCE_FILES})
 
-set(DEPLIBS ${platform_LIBRARIES} tsreader)
+set(DEPLIBS ${p8-platform_LIBRARIES} tsreader)
 
 find_package(JsonCpp REQUIRED)
 list(APPEND DEPLIBS ${JSONCPP_LIBRARIES})
From 1a64ac5e36a90cbe2c025003bb25aca5390d2ef1 Mon Sep 17 00:00:00 2001
From: "h.udo" <hudokkow@gmail.com>
Date: Wed, 6 Jan 2016 15:08:31 +0000
Subject: [PATCH] [src] Update to p8-platform

---
 src/EventsThread.cpp                 |  2 +-
 src/EventsThread.h                   |  4 ++--
 src/KeepAliveThread.cpp              |  4 ++--
 src/KeepAliveThread.h                |  4 ++--
 src/argustvrpc.cpp                   | 12 ++++++------
 src/client.h                         |  2 +-
 src/lib/tsreader/FileReader.cpp      |  2 +-
 src/lib/tsreader/FileReader.h        |  2 +-
 src/lib/tsreader/MultiFileReader.cpp | 10 +++++-----
 src/lib/tsreader/TSReader.cpp        |  2 +-
 src/lib/tsreader/TSReader.h          |  2 +-
 src/pvrclient-argustv.cpp            |  6 +++---
 src/pvrclient-argustv.h              |  4 ++--
 src/tools.cpp                        |  4 ++--
 src/tools.h                          |  2 +-
 src/utils.cpp                        |  2 +-
 src/utils.h                          |  4 ++--
 17 files changed, 34 insertions(+), 34 deletions(-)

diff --git a/src/EventsThread.cpp b/src/EventsThread.cpp
index eaa7cd4..821a437 100644
--- a/src/EventsThread.cpp
+++ b/src/EventsThread.cpp
@@ -86,7 +86,7 @@ void *CEventsThread::Process()
         if (events.size() > 0u) HandleEvents(events);
       }
     }
-    // The new PLATFORM:: thread library has a problem with stopping a thread that is doing a long sleep
+    // The new P8PLATFORM:: thread library has a problem with stopping a thread that is doing a long sleep
     for (int i = 0; i < 100; i++)
     {
       if (Sleep(100)) break;
diff --git a/src/EventsThread.h b/src/EventsThread.h
index d876401..05dabfb 100644
--- a/src/EventsThread.h
+++ b/src/EventsThread.h
@@ -19,9 +19,9 @@
  *
  */
 
-#include "platform/threads/threads.h"
+#include "p8-platform/threads/threads.h"
 
-class CEventsThread : public PLATFORM::CThread
+class CEventsThread : public P8PLATFORM::CThread
 {
 public:
   CEventsThread(void);
diff --git a/src/KeepAliveThread.cpp b/src/KeepAliveThread.cpp
index e704f40..e8a6cbc 100644
--- a/src/KeepAliveThread.cpp
+++ b/src/KeepAliveThread.cpp
@@ -18,7 +18,7 @@
  *
  */
 
-#include "platform/os.h"
+#include "p8-platform/os.h"
 #include "client.h" //for XBMC->Log
 #include "utils.h"
 #include "argustvrpc.h"
@@ -43,7 +43,7 @@ void *CKeepAliveThread::Process()
   {
     int retval = ArgusTV::KeepLiveStreamAlive();
     XBMC->Log(LOG_DEBUG, "CKeepAliveThread:: KeepLiveStreamAlive returned %i", (int) retval);
-    // The new PLATFORM:: thread library has a problem with stopping a thread that is doing a long sleep
+    // The new P8PLATFORM:: thread library has a problem with stopping a thread that is doing a long sleep
     for (int i = 0; i < 100; i++)
     {
       if (Sleep(100)) break;
diff --git a/src/KeepAliveThread.h b/src/KeepAliveThread.h
index c637e04..d9946d1 100644
--- a/src/KeepAliveThread.h
+++ b/src/KeepAliveThread.h
@@ -16,9 +16,9 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "platform/threads/threads.h"
+#include "p8-platform/threads/threads.h"
 
-class CKeepAliveThread : public PLATFORM::CThread
+class CKeepAliveThread : public P8PLATFORM::CThread
 {
 public:
   CKeepAliveThread();
diff --git a/src/argustvrpc.cpp b/src/argustvrpc.cpp
index 25ab115..88a4c7c 100644
--- a/src/argustvrpc.cpp
+++ b/src/argustvrpc.cpp
@@ -25,13 +25,13 @@
 
 #include <stdio.h>
 #include <sys/stat.h>
-#include "platform/os.h"
+#include "p8-platform/os.h"
 #include "client.h"
 #include "pvrclient-argustv.h"
 #include "utils.h"
 #include "argustvrpc.h"
-#include "platform/threads/mutex.h"
-#include "platform/util/StdString.h"
+#include "p8-platform/threads/mutex.h"
+#include "p8-platform/util/StdString.h"
 
 using namespace ADDON;
 
@@ -43,7 +43,7 @@ using namespace ADDON;
  */
 namespace ArgusTV
 {
-  PLATFORM::CMutex communication_mutex;
+  P8PLATFORM::CMutex communication_mutex;
 
   /**
    * \brief Do some internal housekeeping at the start
@@ -65,7 +65,7 @@ namespace ArgusTV
 
   int ArgusTVRPC(const std::string& command, const std::string& arguments, std::string& json_response)
   {
-    PLATFORM::CLockObject critsec(communication_mutex);
+    P8PLATFORM::CLockObject critsec(communication_mutex);
     std::string url = g_szBaseURL + command;
     int retval = E_FAILED;
     XBMC->Log(LOG_DEBUG, "URL: %s\n", url.c_str());
@@ -98,7 +98,7 @@ namespace ArgusTV
 
   int ArgusTVRPCToFile(const std::string& command, const std::string& arguments, std::string& filename, long& http_response)
   {
-    PLATFORM::CLockObject critsec(communication_mutex);
+    P8PLATFORM::CLockObject critsec(communication_mutex);
     std::string url = g_szBaseURL + command;
     int retval = E_FAILED;
     XBMC->Log(LOG_DEBUG, "URL: %s writing to file %s\n", url.c_str(), filename.c_str());
diff --git a/src/client.h b/src/client.h
index dc7d696..ef05bae 100644
--- a/src/client.h
+++ b/src/client.h
@@ -23,7 +23,7 @@
 #ifndef CLIENT_H
 #define CLIENT_H
 
-#include "platform/util/StdString.h"
+#include "p8-platform/util/StdString.h"
 #include "kodi/libXBMC_addon.h"
 #include "kodi/libXBMC_pvr.h"
 
diff --git a/src/lib/tsreader/FileReader.cpp b/src/lib/tsreader/FileReader.cpp
index 50b754a..7f34f36 100644
--- a/src/lib/tsreader/FileReader.cpp
+++ b/src/lib/tsreader/FileReader.cpp
@@ -35,7 +35,7 @@
 #include "FileReader.h"
 #include "client.h" //for XBMC->Log
 #include <algorithm> //std::min, std::max
-#include "platform/util/timeutils.h" // for usleep
+#include "p8-platform/util/timeutils.h" // for usleep
 
 using namespace ADDON;
 
diff --git a/src/lib/tsreader/FileReader.h b/src/lib/tsreader/FileReader.h
index 00472b6..8a2811b 100644
--- a/src/lib/tsreader/FileReader.h
+++ b/src/lib/tsreader/FileReader.h
@@ -33,7 +33,7 @@
  *  http://forums.dvbowners.com/
  */
 
-#include "platform/os.h"
+#include "p8-platform/os.h"
 
 namespace ArgusTV
 {
diff --git a/src/lib/tsreader/MultiFileReader.cpp b/src/lib/tsreader/MultiFileReader.cpp
index 98b3f4d..4802d97 100644
--- a/src/lib/tsreader/MultiFileReader.cpp
+++ b/src/lib/tsreader/MultiFileReader.cpp
@@ -37,19 +37,19 @@
 #include <string>
 #include "utils.h"
 #include <algorithm>
-#include "platform/util/timeutils.h"
-#include "platform/util/StdString.h"
+#include "p8-platform/util/timeutils.h"
+#include "p8-platform/util/StdString.h"
 
 #include <limits.h>
 
 #if !defined(TARGET_WINDOWS)
 #include <sys/time.h>
-#include "platform/os.h"
+#include "p8-platform/os.h"
 #define SUCCEEDED(hr) (((HRESULT)(hr)) >= 0)
 #endif
 
 using namespace ADDON;
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 
 //Maximum time in msec to wait for the buffer file to become available - Needed for DVB radio (this sometimes takes some time)
 #define MAX_BUFFER_TIMEOUT 1500
@@ -123,7 +123,7 @@ namespace ArgusTV
     if (RefreshTSBufferFile() == S_FALSE)
     {
       // For radio the buffer sometimes needs some time to become available, so wait and try it more than once
-      PLATFORM::CTimeout timeout(MAX_BUFFER_TIMEOUT);
+      P8PLATFORM::CTimeout timeout(MAX_BUFFER_TIMEOUT);
 
       do
       {
diff --git a/src/lib/tsreader/TSReader.cpp b/src/lib/tsreader/TSReader.cpp
index 440b976..667445b 100644
--- a/src/lib/tsreader/TSReader.cpp
+++ b/src/lib/tsreader/TSReader.cpp
@@ -31,7 +31,7 @@
 #include "client.h" //for XBMC->Log
 #include "MultiFileReader.h"
 #include "utils.h"
-#include "platform/os.h"
+#include "p8-platform/os.h"
 
 using namespace ADDON;
 
diff --git a/src/lib/tsreader/TSReader.h b/src/lib/tsreader/TSReader.h
index 90110f8..aaf644e 100644
--- a/src/lib/tsreader/TSReader.h
+++ b/src/lib/tsreader/TSReader.h
@@ -30,7 +30,7 @@
 
 #include "client.h"
 #include "FileReader.h"
-#include "platform/util/StdString.h"
+#include "p8-platform/util/StdString.h"
 
 namespace ArgusTV
 {
diff --git a/src/pvrclient-argustv.cpp b/src/pvrclient-argustv.cpp
index cca75a9..de85658 100644
--- a/src/pvrclient-argustv.cpp
+++ b/src/pvrclient-argustv.cpp
@@ -29,15 +29,15 @@
 #include "utils.h"
 #include "pvrclient-argustv.h"
 #include "argustvrpc.h"
-#include "platform/util/timeutils.h"
-#include "platform/util/StdString.h"
+#include "p8-platform/util/timeutils.h"
+#include "p8-platform/util/StdString.h"
 
 #include "lib/tsreader/TSReader.h"
 
 using namespace std;
 using namespace ADDON;
 
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 using namespace ArgusTV;
 
 #define SIGNALQUALITY_INTERVAL 10
diff --git a/src/pvrclient-argustv.h b/src/pvrclient-argustv.h
index ff776a4..5d4a502 100644
--- a/src/pvrclient-argustv.h
+++ b/src/pvrclient-argustv.h
@@ -17,7 +17,7 @@
  *
  */
 
-#include "platform/os.h"
+#include "p8-platform/os.h"
 
 #include <vector>
 
@@ -126,7 +126,7 @@ class cPVRClientArgusTV
   time_t                  m_BackendUTCoffset;
   time_t                  m_BackendTime;
 
-  PLATFORM::CMutex        m_ChannelCacheMutex;
+  P8PLATFORM::CMutex        m_ChannelCacheMutex;
   std::vector<cChannel*>   m_TVChannels; // Local TV channel cache list needed for id to guid conversion
   std::vector<cChannel*>   m_RadioChannels; // Local Radio channel cache list needed for id to guid conversion
   int                     m_epg_id_offset;
diff --git a/src/tools.cpp b/src/tools.cpp
index 3b2835f..48507d5 100644
--- a/src/tools.cpp
+++ b/src/tools.cpp
@@ -24,10 +24,10 @@
  */
 
 #include "tools.h"
-#include "platform/util/timeutils.h"
+#include "p8-platform/util/timeutils.h"
 
 using namespace ADDON;
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 
 // --- cTimeMs ---------------------------------------------------------------
 
diff --git a/src/tools.h b/src/tools.h
index 392e7fd..f5be097 100644
--- a/src/tools.h
+++ b/src/tools.h
@@ -23,7 +23,7 @@
 #define __TOOLS_H
 
 #include "client.h"
-#include "platform/util/StdString.h"
+#include "p8-platform/util/StdString.h"
 #include <errno.h>
 #include <fcntl.h>
 #include <stddef.h>
diff --git a/src/utils.cpp b/src/utils.cpp
index 0c10eab..68a4f1b 100644
--- a/src/utils.cpp
+++ b/src/utils.cpp
@@ -20,7 +20,7 @@
 #endif
 
 #include "utils.h"
-#include "platform/os.h"
+#include "p8-platform/os.h"
 #include "client.h" //For XBMC->Log
 #include <string>
 #include <algorithm> // sort
diff --git a/src/utils.h b/src/utils.h
index dfdedc2..cfaa91c 100644
--- a/src/utils.h
+++ b/src/utils.h
@@ -19,8 +19,8 @@
 
 #include <string>
 #include <json/json.h>
-#include "platform/util/StdString.h"
-#include "platform/util/util.h"
+#include "p8-platform/util/StdString.h"
+#include "p8-platform/util/util.h"
 
 // don't use the UNUSED macro from StdString.h as it will generate "error: statement has no effect [-Werror=unused-value]"
 #define NOTUSED(x) ((void) x)
