From e3d6cb6a4c8829ff841fab966c8e5c8385b82350 Mon Sep 17 00:00:00 2001
From: "h.udo" <hudokkow@gmail.com>
Date: Wed, 6 Jan 2016 15:21:54 +0000
Subject: [PATCH] [cmake] Update to p8-platform

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 85e3767..a1bce83 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,7 @@ enable_language(CXX)
 
 find_package(kodi REQUIRED)
 find_package(kodiplatform REQUIRED)
-find_package(platform REQUIRED)
+find_package(p8-platform REQUIRED)
 
 add_definitions(-DUSE_DEMUX)
 
@@ -68,7 +68,7 @@ endif()
 
 include_directories(${INCLUDES}
                     ${kodiplatform_INCLUDE_DIRS}
-                    ${platform_INCLUDE_DIRS}
+                    ${p8-platform_INCLUDE_DIRS}
                     ${KODI_INCLUDE_DIR})
 
 build_addon(pvr.vdr.vnsi VDR DEPLIBS)


From 20654cb4defa3bd201c03f65ae0a48f00b37cb5c Mon Sep 17 00:00:00 2001
From: "h.udo" <hudokkow@gmail.com>
Date: Wed, 6 Jan 2016 15:09:10 +0000
Subject: [PATCH] [src] Update to p8-platform

---
 src/VNSIAdmin.cpp      | 2 +-
 src/VNSIAdmin.h        | 2 +-
 src/VNSIData.cpp       | 4 ++--
 src/VNSIData.h         | 6 +++---
 src/VNSISession.cpp    | 6 +++---
 src/VNSISession.h      | 8 ++++----
 src/client.cpp         | 6 +++---
 src/requestpacket.cpp  | 2 +-
 src/responsepacket.cpp | 2 +-
 9 files changed, 19 insertions(+), 19 deletions(-)

diff --git a/src/VNSIAdmin.cpp b/src/VNSIAdmin.cpp
index d5b5887..f2c2192 100644
--- a/src/VNSIAdmin.cpp
+++ b/src/VNSIAdmin.cpp
@@ -135,7 +135,7 @@ CVisGUIShader *vis_shader = NULL;
 
 
 using namespace ADDON;
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 
 
 class cOSDTexture
diff --git a/src/VNSIAdmin.h b/src/VNSIAdmin.h
index 3c523a4..eeda539 100644
--- a/src/VNSIAdmin.h
+++ b/src/VNSIAdmin.h
@@ -89,5 +89,5 @@ class cVNSIAdmin : public cVNSIData
   int m_width, m_height;
   int m_osdWidth, m_osdHeight;
   cOSDRender *m_osdRender;
-  PLATFORM::CMutex m_osdMutex;
+  P8PLATFORM::CMutex m_osdMutex;
 };
diff --git a/src/VNSIData.cpp b/src/VNSIData.cpp
index 9a0c51c..8f442c0 100644
--- a/src/VNSIData.cpp
+++ b/src/VNSIData.cpp
@@ -23,10 +23,10 @@
 #include "responsepacket.h"
 #include "requestpacket.h"
 #include "vnsicommand.h"
-#include "platform/util/StdString.h"
+#include "p8-platform/util/StdString.h"
 
 using namespace ADDON;
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 
 cVNSIData::SMessage &
 cVNSIData::Queue::Enqueue(uint32_t serial)
diff --git a/src/VNSIData.h b/src/VNSIData.h
index 897cad1..cc147f5 100644
--- a/src/VNSIData.h
+++ b/src/VNSIData.h
@@ -29,7 +29,7 @@
 class cResponsePacket;
 class cRequestPacket;
 
-class cVNSIData : public cVNSISession, public PLATFORM::CThread
+class cVNSIData : public cVNSISession, public P8PLATFORM::CThread
 {
 public:
 
@@ -85,14 +85,14 @@ class cVNSIData : public cVNSISession, public PLATFORM::CThread
 
   struct SMessage
   {
-    PLATFORM::CEvent event;
+    P8PLATFORM::CEvent event;
     std::unique_ptr<cResponsePacket> pkt;
   };
 
   class Queue {
     typedef std::map<int, SMessage> SMessages;
     SMessages m_queue;
-    PLATFORM::CMutex m_mutex;
+    P8PLATFORM::CMutex m_mutex;
 
   public:
     SMessage &Enqueue(uint32_t serial);
diff --git a/src/VNSISession.cpp b/src/VNSISession.cpp
index 0fa4c54..8d87ffa 100644
--- a/src/VNSISession.cpp
+++ b/src/VNSISession.cpp
@@ -31,8 +31,8 @@
 #include "requestpacket.h"
 #include "vnsicommand.h"
 #include "tools.h"
-#include "platform/sockets/tcp.h"
-#include "platform/util/timeutils.h"
+#include "p8-platform/sockets/tcp.h"
+#include "p8-platform/util/timeutils.h"
 
 /* Needed on Mac OS/X */
  
@@ -41,7 +41,7 @@
 #endif
 
 using namespace ADDON;
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 
 cVNSISession::cVNSISession()
   : m_protocol(0)
diff --git a/src/VNSISession.h b/src/VNSISession.h
index 2ece43d..fd0a9b3 100644
--- a/src/VNSISession.h
+++ b/src/VNSISession.h
@@ -22,7 +22,7 @@
 
 #include <stdint.h>
 #include <string>
-#include "platform/threads/threads.h"
+#include "p8-platform/threads/threads.h"
 
 #include <memory>
 
@@ -70,7 +70,7 @@ class cVNSISession
   std::string      m_hostname;
   int              m_port;
   std::string      m_name;
-  PLATFORM::CMutex m_mutex;
+  P8PLATFORM::CMutex m_mutex;
   int              m_protocol;
   std::string      m_server;
   std::string      m_version;
@@ -79,7 +79,7 @@ class cVNSISession
 
   bool readData(uint8_t* buffer, int totalBytes, int timeout);
 
-  PLATFORM::CTcpConnection *m_socket;
-  PLATFORM::CMutex          m_readMutex;
+  P8PLATFORM::CTcpConnection *m_socket;
+  P8PLATFORM::CMutex          m_readMutex;
   bool                      m_connectionLost;
 };
diff --git a/src/client.cpp b/src/client.cpp
index 4ade613..b1de703 100644
--- a/src/client.cpp
+++ b/src/client.cpp
@@ -26,7 +26,7 @@
 #include "VNSIData.h"
 #include "VNSIChannelScan.h"
 #include "VNSIAdmin.h"
-#include "platform/util/util.h"
+#include "p8-platform/util/util.h"
 
 #include <sstream>
 #include <string>
@@ -34,7 +34,7 @@
 
 using namespace std;
 using namespace ADDON;
-using namespace PLATFORM;
+using namespace P8PLATFORM;
 
 ADDON_STATUS m_CurStatus      = ADDON_STATUS_UNKNOWN;
 
@@ -66,7 +66,7 @@ bool IsTimeshift;
 time_t TimeshiftStartTime;
 time_t TimeshiftEndTime;
 time_t TimeshiftPlayTime;
-PLATFORM::CMutex TimeshiftMutex;
+P8PLATFORM::CMutex TimeshiftMutex;
 
 extern "C" {
 
diff --git a/src/requestpacket.cpp b/src/requestpacket.cpp
index 1a4d8fc..4b6e889 100644
--- a/src/requestpacket.cpp
+++ b/src/requestpacket.cpp
@@ -22,7 +22,7 @@
 #include "requestpacket.h"
 #include "vnsicommand.h"
 #include "tools.h"
-#include "platform/sockets/tcp.h"
+#include "p8-platform/sockets/tcp.h"
 
 #include <assert.h>
 #include <stdlib.h>
diff --git a/src/responsepacket.cpp b/src/responsepacket.cpp
index b731271..ecc0422 100644
--- a/src/responsepacket.cpp
+++ b/src/responsepacket.cpp
@@ -22,7 +22,7 @@
 #include "responsepacket.h"
 #include "vnsicommand.h"
 #include "tools.h"
-#include "platform/sockets/tcp.h"
+#include "p8-platform/sockets/tcp.h"
 
 #include <stdexcept>
 #include <stdlib.h>
